src/detect-pcre.c

```diff
@@ -777,7 +776,7 @@ static int DetectPcreParseCapture(const char *regexstr, DetectEngineCtx *de_ctx,
     while (1) {
         SCLogDebug("\'%s\'", regexstr);
 
-        ret = DetectPcreExec(NULL, &parse_capture_regex, regexstr, strlen(regexstr), 0, 0, ov, MAX_SUBSTRINGS);
+        ret = DetectPcreExec(NULL, NULL, &parse_capture_regex, regexstr, strlen(regexstr), 0, 0, ov, MAX_SUBSTRINGS);
```

Comment:
Passing in these NULLs effectively disables the JIT support here, right? Is that the only reason to deal with NULLs to `DetectPcreExec`. If so, it might be cleaner and faster to have dedicated logic for parsing here. That way we can keep the routine used in the packet runtime leaner and cleaner.